packages/backend/src/config.ts

```diff
@@ -45,4 +47,9 @@ export const CONFIG: IConfig = {
     process.env.NEXT_PUBLIC_INFURA_ID,
     '781d8466252d47508e177b8637b1c2fd',
   ),
+  brightIdContext: parseEnv(process.env.BRIGHTID_CONTEXT, 'MetaGame'),
```

Comment:
Does this need to be in the config? Meaning, is it likely to change?

Reply:
it is not likely to change. I suppose it could be set as a constant.